### PR TITLE
Re-add value for Ingress API Version

### PR DESCRIPTION
**Pull request by twz123**
#### What this PR does / why we need it:
It was removed in #11960, but since Kubernetes 1.14, Ingresses live under networking.k8s.io.

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/jenkins/values.yaml [hunk 1]
```diff
@@ -254,6 +254,8 @@ Master:
 
   ingress:
     enabled: false
+    # For Kubernetes v1.14+, use 'networking.k8s.io/v1'
+    ApiVersion: extensions/v1beta1
```
Comment (maorfr) [inline stable/jenkins/values.yaml]: please change to `apiVersion` ;)

Comment (twz123) [reply stable/jenkins/values.yaml]: Tried to stay in sync with the NetworkPolicy stuff... okay to diverge from that?

Comment (maorfr) [reply stable/jenkins/values.yaml]: we are trying to go for a [best practice](https://helm.sh/docs/chart_best_practices/) chart

Comment (twz123) [reply stable/jenkins/values.yaml]: Okie dokie!

**Comment by maorfr** [comment]
/ok-to-test

appreciate it!

**Pull request merged**